app/views/users/edit_phone/choose_delivery_method.html.slim

```diff
@@ -0,0 +1,15 @@
+- title t('titles.edit_info.confirm_phone')
+
+h1.heading = t('forms.phone_confirmation.header_text')
+
+p#2fa-description = t('devise.two_factor_authentication.choose_delivery_confirmation',
+  phone: @phone_number)
+
+= button_to t('devise.two_factor_authentication.buttons.' \
+  'confirm_with_sms'), phone_confirmation_send_path,
+  class: 'btn btn-primary mb2', method: :get, params: { delivery_method: :sms }
+= button_to t('devise.two_factor_authentication.buttons.' \
+  'confirm_with_voice'), phone_confirmation_send_path,
+  class: 'btn btn-primary mb2', method: :get, params: { delivery_method: :voice }
+
+p.mt2.mb0.italic = t('devise.two_factor_authentication.otp_sms_disclaimer')
```

Comment:
I was confused by the placement of this view. Is the `edit_phone` folder the best place for it? Since it's used by multiple controllers, I would maybe put it under a `shared` folder somewhere? Or perhaps instead of using `render`, which doesn't change the URL, create a new controller + view for this delivery selection screen, which can have its own meaningful URL.